sscanss/app/dialogs/managers.py

```diff
@@ -943,3 +947,153 @@ def moveDetectorsButtonClicked(self):
+        """Calculates the memory used by the sample
+
+        :return memory_str: memory consumed in mb
+        :rtype: string
+        """
+        if isinstance(self.parent_model.sample, Mesh):
+            bytes_used = self.parent_model.sample.vertices.nbytes
+        elif isinstance(self.parent_model.sample, Volume):
+            bytes_used = self.parent_model.sample.data.nbytes
+        elif self.parent_model.sample is None:
+            bytes_used = 0
+        memory_str = self._bytesToMb(bytes_used)
+        return memory_str
+
+    def _bytesToMb(self, bytes_used):
+        """Converts the bytes consumed to mb
+
+        :param bytes_used: the bytes used by the sample
+        :type bytes_used: int
+        :return memory: consumed in mb
+        :rtype: string
+        """
+        bytes_to_mb_factor = 1 / (1024**2)
+        memory = bytes_used * bytes_to_mb_factor
+        memory_str = '0' if memory == 0 else f'{memory:.4f}'
+        return memory_str
+
+    def _addVolumePropertyValues(self):
+        """Adds the Volume property values to the table"""
+        memory_str = self._getMemoryUsed()
+        self._add_property_value(memory_str, row=0)
+
+        x, y, z = self.parent_model.sample.data.shape
+        dimension_str = f'{x} x {y} x {z}'
+        self._add_property_value(dimension_str, row=1)
+
+        voxel_size = self.parent_model.sample.voxel_size
+        voxel_size_str = f'[x: {voxel_size[0]},  y: {voxel_size[1]},  z: {voxel_size[2]}]'
+        self._add_property_value(voxel_size_str, row=2)
+
+        min_intensity = np.min(self.parent_model.sample.data)
+        self._add_property_value(min_intensity, row=3)
+
+        max_intensity = np.max(self.parent_model.sample.data)
+        self._add_property_value(max_intensity, row=4)
+
+    def addSamplePropertyValues(self):
+        """Adds the sample property values to the table"""
+        if isinstance(self.parent_model.sample, Mesh):
+            self._addMeshPropertyValues()
+        elif isinstance(self.parent_model.sample, Volume):
+            self._addVolumePropertyValues()
+        elif self.parent_model.sample is None:
+            self._add_property_value('0', row=0)
+
+    def _addItem(self, item, is_label, row):
+        """Adds a Property label or Value to the table
+
+        :param item: the item to add to the table  
+        :type item: Union[string, int]
+        :param is_label: identifies item as either label or value
+        :type is_label: boolean
+        :param row: the row in table to add the item to
+        :type row: int
+        """
+        col = 0 if is_label else 1
+        item = str(item)
+        entry_item = QtWidgets.QTableWidgetItem(item)
+        entry_item.setTextAlignment(QtCore.Qt.AlignCenter)
+        self.sample_property_table.setItem(row, col, entry_item)
+
+    _add_property_label = partialmethod(_addItem, is_label=True)
+    _add_property_value = partialmethod(_addItem, is_label=False)
+
+    def closeEvent(self, event):
```

Comment:
This is not needed. You only need to override the close event handler if you are changing its behaviour like performing some clean-up

Reply:
Unnecessary method removed.